src/commands/build.ts

```diff
@@ -131,10 +168,60 @@ export default class Build extends BaseCommand {
       );
     }
 
+    if (flags.path) {
+      let trackingPlans = [] 
+      for (const plan of configPlans) {
+        const oldPath = './' + plan.path
+        const configPlan = await loadTrackingPlan(this.configPath, plan)
+        const { path } = await this.prompt({
+          type: "input",
+          name: "path",
+          message: this.formatQuestion(
+            `Enter a directory for the Tracking Plan ${chalk.green(
+              configPlan?.name
+            )} output:`,
+            [
+              "The generated client will be stored in this directory.",
+              "Directories will be automatically created, if needed.",
+            ]
+          ),
+          filter: normalize,
+          default: plan?.path
+        });
+
+        plan!.path = path;
+
+        // fetch tracking plan and store in new path
+        let newTrackingPlan: SegmentAPI.TrackingPlan | undefined;
+        try {
+          CliUx.ux.action.status = `Fetching tracking plan ${plan.name}.`;
+          newTrackingPlan = await fetchTrackingPlan(
+            plan.id,
+            flags.token ?? this.apiToken!
+          );
+        } catch (error) {
+          const errorMessage = isWrappedError(error)
+            ? error.description
+            : "API request failed";
+          debug(`${errorMessage}. Using local copy of ${plan.name} instead.`);
+          CliUx.ux.action.status = `Error fetching tracking plan ${plan.name}, using local copy instead.`;
+        }
+        if (newTrackingPlan !== undefined) {
+          CliUx.ux.action.status = `Updating local copy of tracking plan ${plan.name}.`;
+          // Update plan.json with the latest Tracking Plan.
+          await writeTrackingPlan(this.configPath, newTrackingPlan, plan);
+          CliUx.ux.action.status = `Plan ${plan.name} updated.`;
+        }
+        trackingPlans.push({name: configPlan?.name, path: plan.path, id: plan.id})
+        this.deleteFolder(oldPath)
+      }
+      await saveWorkspaceConfig({...this.workspaceConfig, trackingPlans}, this.configPath);
```

Comment:
This is almost the same logic as `loadTrackingPlans`, do we actually need this? 

Also this might cause to "double update" the TP if you pass in the `--update` flag doesn't it? as this always fetches the TP regardless of the flags.